schemas/rum/_perf-info-schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "rum/_perf-info-schema.json",
+  "title": "RumPerfInfo",
+  "type": "object",
+  "description": "Schema of properties for a technical performance metic",
+  "required": ["view_min", "view_max", "view_average"],
+  "properties": {
+    "view_min": {
+      "type": "number",
+      "description": "The minimum value seen for this metic during the view's lifetime.",
+      "readOnly": true
+    },
+    "view_max": {
+      "type": "number",
+      "description": "The maximum value seen for this metic during the view's lifetime.",
+      "readOnly": true
+    },
+    "view_average": {
+      "type": "number",
+      "description": "The minimum value seen for this metic during the view's lifetime.",
+      "readOnly": true
+    },
+    "metric_max": {
```

Comment:
Good suggestion! I'm still unhappy with this name though. It still sounds like this is the max we saw in the session, rather than the maximum possible value....